cli/ts/commands/deployPoll.ts

```diff
@@ -105,62 +100,17 @@ export const deployPoll = async (
 
   const unserializedKey = PubKey.deserialize(coordinatorPubkey);
 
-  // get the poseidon contracts addresses
-  const poseidonT3 = readContractAddress("PoseidonT3");
-  const poseidonT4 = readContractAddress("PoseidonT4");
-  const poseidonT5 = readContractAddress("PoseidonT5");
-  const poseidonT6 = readContractAddress("PoseidonT6");
-
   // get the verifier contract
   const verifierContractAddress = readContractAddress("Verifier");
 
-  // deploy the message processor
-  const messageProcessorContract = await deployMessageProcessor(
-    verifierContractAddress,
-    vkRegistry,
-    poseidonT3,
-    poseidonT4,
-    poseidonT5,
-    poseidonT6,
-    signer,
-    true,
-  );
-
-  // deploy the tally contract
-  const tallyContract = await deployTally(
-    verifierContractAddress,
-    vkRegistry,
-    poseidonT3,
-    poseidonT4,
-    poseidonT5,
-    poseidonT6,
-    signer,
-    true,
-  );
-
-  // deploy the subsidy contract
-  const subsidyContract = await deploySubsidy(
-    verifierContractAddress,
-    vkRegistry,
-    poseidonT3,
-    poseidonT4,
-    poseidonT5,
-    poseidonT6,
-    signer,
-    true,
-  );
-
   const maciAbi = parseArtifact("MACI")[0];
   const maciContract = new BaseContract(maci, maciAbi, signer) as MACI;
 
   // deploy the poll
   let pollAddr = "";
-
-  const [messageProcessorContractAddress, tallyContractAddress, subsidyContractAddress] = await Promise.all([
-    messageProcessorContract.getAddress(),
-    tallyContract.getAddress(),
-    subsidyContract.getAddress(),
-  ]);
+  let messageProcessorContractAddress = "";
+  let tallyContractAddress = "";
+  let subsidyContractAddress = "";
```

Comment:
we should make this optional, so to be changed in the interface `PollContracts` to be optional

Reply:
Make what to be optional? `subsidyContractAddress` only or all of them?

Reply:
subsidy, as we won't always deploy it